core/pom.xml

```diff
@@ -394,7 +401,8 @@
 					</execution>
 				</executions>
 				<configuration>
-					<source>8</source>
+					<release>17</release>
```

Comment:
**suggestion:** Good catch, I think you could use the property instead :

```suggestion
					<release>${maven.compiler.source}</release>
```

Reply:
Or maybe (and then creating necessary properties) 

```suggestion
<source>${maven.javadoc.source}</source>
<release>${maven.javadoc.release}</release>
```

Reply:
Good point, I will resort to maven.compiler.source, think that makes sense.